### PR TITLE
Nodejs packages workflow

### PR DESCRIPTION
**Pull request by Thoralf-M**

### CODE REVIEW

File: .github/workflows/covector-version-or-publish.yml [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        if: matrix.os == 'windows-latest'
+        run: |
+          $ProgressPreference = 'SilentlyContinue'
+          Invoke-WebRequest https://iotaledger-files.s3.eu-central-1.amazonaws.com/prebuild/openssl/windows/vcpkg-export-openssl-1.1.1g.zip -OutFile openssl.zip
+          Expand-Archive openssl.zip
+          Remove-Item openssl.zip
+          echo "OPENSSL_ROOT_DIR=${{ github.workspace }}/openssl/installed/x64-windows-static" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+      - name: Install dependencies
+        run: yarn
+        working-directory: bindings/nodejs
```
Comment (rajivshah3) [inline .github/workflows/covector-version-or-publish.yml]: We're no longer using paho so this isn't necessary, right?

Comment (Thoralf-M) [reply .github/workflows/covector-version-or-publish.yml]: Yes

File: .changes/readme.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+As you create PRs and make changes that require a version bump, please add a new markdown file in this folder. You do not note the version _number_, but rather the type of bump that you expect: major, minor, or patch. The filename is not important, as long as it is a `.md`, but we recommend it represents the overall change for our sanity.
+
+When you select the version bump required, you do _not_ need to consider dependencies. Only note the package with the actual change, and any packages that depend on that package will be bumped automatically in the process.
+
+Use the following format:
+
+```md
+---
+"tauri.js": patch
+"tauri": minor
```
Comment (rajivshah3) [inline .changes/readme.md]:
```suggestion
"nodejs-binding": patch
"python-binding": minor
```

File: .github/workflows/covector-version-or-publish.yml [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+      - name: Get current date
+        if: matrix.os == 'windows-latest'
+        run: echo "CURRENT_DATE=$(Get-Date -Format "yyyy-MM-dd")" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+
+      - name: Install required packages (Ubuntu)
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt-get update
+          sudo apt-get install libudev-dev libusb-1.0-0-dev
```
Comment (rajivshah3) [inline .github/workflows/covector-version-or-publish.yml]:
```suggestion
```

This was necessary for the Ledger lib but I don't think you need these

File: .github/workflows/covector-version-or-publish.yml [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
```
Comment (rajivshah3) [inline .github/workflows/covector-version-or-publish.yml]:
```suggestion
```

File: bindings/nodejs/package.json [hunk 5]
```diff
@@ -10,13 +10,19 @@
...
+    "install": "prebuild-install --tag-prefix nodejs-binding-v || npm run rebuild",
+    "build:neon": "neon build --release && node scripts/move-lib.js",
+    "rebuild": "npm run build:neon && node scripts/strip.js",
+    "prebuild:node": "prebuild --prepack scripts/node-neon-build.js --strip",
+    "prebuild:electron": "node scripts/electron-prebuild.js",
+    "build:docs": "./node_modules/.bin/jsdoc lib/index.js",
     "test": "mocha ./tests --exclude assertions.js --timeout 0"
   },
   "devDependencies": {
     "mocha": "^8.2.1"
```
Comment (rajivshah3) [inline bindings/nodejs/package.json]: `electron-build-env` and `prebuild` need to be added as well

**Review by rajivshah3 — CHANGES_REQUESTED** [review]

**Review by rajivshah3 — APPROVED** [review]

**Pull request merged**